mlflow/tracking/fluent.py

```diff
@@ -1020,3 +1036,26 @@ def _get_experiment_id():
         or _get_experiment_id_from_env()
         or (is_in_databricks_notebook() and get_notebook_id())
     ) or deprecated_default_exp_id
+
+
+def universal_autolog(
+    log_input_example=False, log_model_signature=True
+):  # pylint: disable=unused-argument
```

Comment:
Do we plan to add docs to this method as part of a follow-up PR? (I know there's a JIRA ticket for this, but wanted to double check)

Reply:
yup! I have a branch with some docs so far, just needs to be rebased on these new changes